assets/overlays/aws-ebs/generated/hypershift/controller.yaml

```
@@ -414,6 +417,43 @@ spec:
         - mountPath: /etc/hosted-kubernetes
           name: hosted-kubeconfig
           readOnly: true
+      initContainers:
+      - command:
+        - sh
+        - -c
+        - |
+          # Define file path variables
+          CREDENTIALS_FILE=/var/run/aws/keys/credentials
+          AUTH_CREDENTIALS_FILE=/var/run/aws/auth/credentials
+          AWS_ACCESS_KEY_ID_FILE=/var/run/aws/keys/aws_access_key_id
+          AWS_SECRET_ACCESS_KEY_FILE=/var/run/aws/keys/aws_secret_access_key
+
+          # If credentials key exists in ebs-cloud-credentials secret, then use it as the auth file
+          if [ -e "$CREDENTIALS_FILE" ]; then
+              cp "$CREDENTIALS_FILE" "$AUTH_CREDENTIALS_FILE"
+              exit 0
```

nit: it would be nice to log what the init container did - now its log is totally empty.

```suggestion
              echo "Kubernetes Secret already contains credentials file, copied to the right place: $AUTH_CREDENTIALS_FILE"
              exit 0
```